baselines/fedavgm/fedavgm/models.py

```diff
@@ -0,0 +1,121 @@
+"""CNN model architecture."""
+
+from flwr.common import ndarrays_to_parameters
+from keras.optimizers import SGD
+from keras.regularizers import l2
+from tensorflow import keras
+from tensorflow.nn import local_response_normalization
+
+
+def cnn(input_shape, num_classes):
+    """CNN Model from (McMahan et. al., 2017).
+
+    Communication-efficient learning of deep networks from decentralized data
+    """
+    input_shape = tuple(input_shape)
+
+    weight_decay = 0.004
+    model = keras.Sequential(
+        [
+            keras.layers.Conv2D(
+                64,
+                (5, 5),
+                padding="same",
+                activation="relu",
+                input_shape=input_shape,
+            ),
+            keras.layers.MaxPooling2D((3, 3), strides=(2, 2)),
+            keras.layers.BatchNormalization(),
```

Comment:
Is that ok to focus more on the non-iid behavior (FedAvgM outperforming FedAvg for greater non-iid), instead of the same accuracy level reported on CIFAR-10? I tried multiple models on a centralized setup and none of them presented this ~0.8 accuracy (a more complex model would solve this). In addition to the references mentioned in the README.md, i also checked these (starred from PapersWithCode):
https://github.com/AshwinRJ/Federated-Learning-PyTorch/blob/master/src/models.py#L46-L59
https://github.com/mikudehuane/Federated-Averaging/blob/master/models.py#L48-L61